draft-packages/menu/KaizenDraft/Menu/MenuDropdown.tsx

```diff
@@ -14,20 +14,27 @@ type MenuDropdownProps = {
   } | null
   align?: "left" | "right"
   width?: "default" | "contain"
+  autoHideOnClick?: boolean
```

Comment:
I'm thinking, to minimise the risk of breaking changes, this should be a string of type `"on" | "click-outside-only"`. This way, if we wanted to completely disable the autohide functionality (ie. add an extra option for `"off"`), we can do so, without introducing a breaking change.

Reply:
Or, if you have the time, you could add that `off` option yourself whilst you're in the code